data-es6.js

```diff
@@ -18193,6 +18193,7 @@ exports.tests = [
         ie11: true,
         firefox2: true,
         chrome5: true,
+        deno1_3: false,
```

Comment:
It was removed in [0.37](https://github.com/denoland/deno/releases/tag/v0.37.0), so I think in the scope of this PR it can be marked as from 1.0.